src/utils_format_graphite.c

```diff
@@ -34,6 +34,37 @@
 /* Utils functions to format data sets in graphite format.
  * Largely taken from write_graphite.c as it remains the same formatting */
 
+/* helper function for reverse_hostname */
+void reverse_string(char *r_host, int len) {
+  char t;
+  for (int i = 0; i < len / 2; i++) {
```

Comment:
Just a stylistic comment:
Index `j` could be declared and initialized in the `for` statement so that you can write:
`for (int i = 0, j = len - 1; i <= j; i++, j--) {`